.github/.cspell/dictionary.txt

```diff
@@ -0,0 +1,21 @@
+arjo
+clippy
+deque
+Maerz
+nalgebra
+nanos
+nsec
+nsecs
+println
+rosmsg
+rosrust
+rustc
+rustdocflags
+rustflags
+rustfmt
+rustros
+rustup
+slerp
+tflist
+thiserror
+timetravel
```

Comment:
Could you rename the code?
* tflist -> tf_list
* timetravel -> time_travel

Reply:
Done